awcodify/phoaw
# User authentication guardian

## What's this PR do ?
user authentication using Guardian
## What are the relevant tickets ?
#6
## Where should the reviewer start ?
`session_controller.ex`
## How should this be manually tested ?
1. Open url `/sessions/new`
2. Fill login form and submit
3. You should be redirected to `/users`
## Any background context you want to provide ?
## Deployment notes
## Screenshots (if appropriate)
## Definition of Done:
- [x ] Does this PR require Database changes ?
- [ v] Does this add new dependencies? (network setup, new 3rd party services)
  Guardian
- [ x] Will this feature require a new piece of infrastructure to be implemented?
